source/index.html

```diff
@@ -117,6 +125,14 @@ <h2 class="visually-hidden">Интересная информация</h2>
     <section class="reviews">
       <div class="reviews__wrapper wrapper">
         <h2 class="visually-hidden">Отзывы</h2>
+        <div class="reviews__arrows arrow-switch">
+          <button class="arrow-switch__buttons arrow-switch__buttons--back">
```

Comment:
Почему множественное число, а не единственное?
```suggestion
          <button class="arrow-switch__button arrow-switch__button--back">
```

Reply:
ну их же две)
исправила на единственное

Reply:
Но это же элемент, который одна из них